NAMESPACE

```diff
@@ -128,11 +127,12 @@ export(UserApi)
 export(Writer)
 export(array_info)
 export(compute)
+export(compute_sequentially)
 export(configure)
 export(delayed)
 export(delayedArrayUDF)
 export(delayedSQL)
-export(delayed_args)
+export(delayedUDF)
```

Comment:
Should these be adjusted to `snake_case`?

Reply:
@thetorpedodog really good point. A few things:

* AFAICT in the R world there's not one true way, compared to say Java which is camelCaseAllTheWay, EXCEPT_FOR_CONSTANTS. Although I do think that more often generics/functions (non-methods) are snake_cased, and more often classes are camelCased, with ClassNamesStartingWithAnUppercaseLetter ...
* I imitated `delayedUDF` and `delayedSQL` directly from `TileDB-Cloud-Py`; maybe not the right thing to do ... note that Python has `DelayedUDF` and I chose `delayedUDF` since I wanted these to definitely not look like classes.
* `delayed_args` I took from a generic-name pattern

All those are reasonable individually, perhaps, but taken together they do indeed seem clashy. :^/

Since `delayedUDF` and `delayedSQL` are non-method functions, maybe better to do `delayed_UDF` and `delayed_SQL` -- ? IDK if @eddelbuettel and/or @ihnorton have insight here ...

Reply:
@thetorpedodog one source of comparison is https://github.com/TileDB-Inc/TileDB-R/blob/master/NAMESPACE -- the exports from `TileDB-R` which is maintained by our own expert @eddelbuettel . (Although admittedly many of the names are carry-forwards from TileDB core so maybe some of these are second-hand C++ naming conventions.) Anyway, what I see there is mostly snake_case so I think I should go with that here as well.

Reply:
https://github.com/TileDB-Inc/TileDB-Cloud-R/pull/70

Reply:
... is merged & this is now rebased atop that. :)